zuul.d/nodeset.yaml

```diff
@@ -279,6 +248,16 @@
           - overcloud-novacompute-0
           - overcloud-novacompute-1
           - overcloud-novacompute-2
+      - name: tripleo_controllers
+        nodes:
+          - overcloud-controller-0
+          - overcloud-controller-1
+          - overcloud-controller-2
+      - name: tripleo_computes
+        nodes:
+          - overcloud-novacompute-0
+          - overcloud-novacompute-1
+          - overcloud-novacompute-2
```

Comment:
ack @cescgina on adding these groups for the nova cells but that work is still in progress. I have to check how Bogdan is dealing with those 'fetch compute node hostname' tasks in his patches

so for now i am not going to update

Reply:
oh ok, if there is duplication there it should be fine, I'm approving this change then

Reply:
This should not change anything from the cells work, in the sense that you can continue to use delegate_to and target specific nodes (as Bogdan does in https://review.rdoproject.org/r/c/rdo-jobs/+/53192/39/playbooks/data_plane_adoption/setup_tripleo_hostnamemap.yaml for example)